privacyidea/static/components/audit/controllers/auditControllers.js

```diff
@@ -128,10 +116,4 @@ myApp.controller("auditController", function (AuditFactory, $scope,
         return true;
     };
```

Comment:
I'm not really sure, but I think ``openDate`` can be removed too because it is unused now?

Reply:
I think you are right. Removed it in d2c62ab